src/js/stores/validators/AppFormValidators.js

```diff
@@ -60,7 +60,7 @@ const AppFormValidators = {
       }),
 
   cpus: (value) => !Util.isStringAndEmpty(value) &&
-    !!value.toString().match(/^[0-9\.]+$/),
+    !!value.toString().match(/^[0-9\.]+$/) && value >= 0.01,
```

Comment:
Good point, let's add `parseFloat` for some extra safety – I thought it's already covered by `Util.isStringAndEmpty` which is kinda misleading.